src/python/pants/backend/codegen/export_codegen_goal.py

```diff
@@ -67,8 +68,12 @@ async def export_codegen(
             }
         )
         logger.warning(
-            "No codegen files/targets matched. All codegen target types: "
-            f"{', '.join(codegen_targets)}"
```

Comment:
There's other 2-liners I didn't change because I wanted to fish for feedback,

Reply:
I'm not following what the difference is?

Reply:
Not a difference, just pointing out I softwrapped something only spanning two lines in source. I very much dislike trailing space in the strings (I prefer leading space when nevessary) so I like it. but wanted to see what others thought.

Reply:
Ah I see. I personally think it's worse to go to this because the extra nesting and new lines. Not bad enough that I'll ask people to not use softwrap in their PRs for small lines like this, but I personally don't think I'd want to use it until maybe 3+ lines? (Or that the rest of the code in the block is using it, i.e. `help`)